articles/quickstart/native/ios-objc/_includes/_dependency_centralized.md

```diff
@@ -0,0 +1,26 @@
+## Install
+
+#### Carthage
+
+If you are using Carthage, add the following lines to your `Cartfile`:
+
+```ruby
+github "auth0/Auth0.swift" ~> 1.5
```

Comment:
Can we use `1.+` so it gets always the latest version? This will help to avoid updating this doc with each version change. Also we can add a legend like "check the repo or the "carthage site" to see which is the latest version. If we follow semver we shouldn't have any problem.